build.gradle

```diff
@@ -27,6 +27,15 @@ allprojects {
  */
 allprojects { project ->
 
+    //Skip Javadoc generation for Java 1.8 as it breaks build
+    if (JavaVersion.current().isJava8Compatible()) {
+        allprojects {
```

Comment:
Since this references allprojects, I think it should be outside of the outer allprojects block rather than nested.